test/e2e/storage/testsuites/ephemeral.go

```diff
@@ -213,6 +213,67 @@ func (p *ephemeralTestSuite) DefineTests(driver storageframework.TestDriver, pat
 		l.testCase.TestEphemeral()
 	})
 
+	ginkgo.It("should support expansion of pvcs created for ephemeral pvcs", func() {
+		if pattern.VolType != storageframework.GenericEphemeralVolume {
+			e2eskipper.Skipf("Skipping %s test for expansion", pattern.VolType)
+		}
+
+		init()
+		defer cleanup()
+
+		if !driver.GetDriverInfo().Capabilities[storageframework.CapOnlineExpansion] {
+			e2eskipper.Skipf("Driver %q does not support online volume expansion - skipping", driver.GetDriverInfo().Name)
+		}
+
+		l.testCase.ReadOnly = false
+		l.testCase.RunningPodCheck = func(pod *v1.Pod) interface{} {
+			podName := pod.Name
+			framework.Logf("Running volume expansion checks %s", podName)
+
+			outerPodVolumeSpecName := ""
+			for i := range pod.Spec.Volumes {
+				volume := pod.Spec.Volumes[i]
+				if volume.Ephemeral != nil {
+					outerPodVolumeSpecName = volume.Name
+					break
+				}
+			}
+			pvcName := fmt.Sprintf("%s-%s", podName, outerPodVolumeSpecName)
+			pvc, err := f.ClientSet.CoreV1().PersistentVolumeClaims(pod.Namespace).Get(context.TODO(), pvcName, metav1.GetOptions{})
+			framework.ExpectNoError(err, "error getting ephemeral pvc")
+
+			ginkgo.By("Expanding current pvc")
+			currentPvcSize := pvc.Spec.Resources.Requests[v1.ResourceStorage]
+			newSize := currentPvcSize.DeepCopy()
+			newSize.Add(resource.MustParse("1Gi"))
+			framework.Logf("currentPvcSize %v, newSize %v", currentPvcSize, newSize)
```

Comment:
nit: is this the new size already? Would "requested new size" be better here?

Reply:
fixed